acceptancetests/assess_caas_bootstrap.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python
+""" Test caas k8s cluster bootstrap
+"""
+
+from __future__ import print_function
+
+import argparse
+import logging
+import sys
+import os
+
+from deploy_stack import (
+    BootstrapManager,
+)
+from utility import (
+    add_basic_testing_arguments,
+    configure_logging,
+)
+
+from jujucharm import (
+    local_charm_path
+)
+
+
+__metaclass__ = type
+
+
+log = logging.getLogger("assess_caas_bootstrap")
+
+
+def assess_caas_bootstrap(client):
+    # Deploy charms, there are several under ./repository
```

Comment:
I don't think this comment is true - we're not deploying charms in here but bootstrapping a k8s cluster

Reply:
yes, there is a lint target in `Makefile` now, It will need to be added to the jenkins stage